kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/api.js

```diff
@@ -43,34 +47,23 @@ export function fetchAddresses(withChannelId = '') {
   });
 }
 
-export function fetchDevices(withChannelId = '') {
```

Comment:
This is abstracted into three nearly identical interfaces for fetching static, dynamic and generic `NetworkLocationResource`s